live-image/files/usr/local/sbin/dj_live

```diff
@@ -63,9 +63,9 @@ EOF
 
 		echo "Updating DOMjudge web interface admin user password..."
 		if [ -z "$PASS" ]; then
-			read -es -p "Enter new admin user password: "  PASS1 ; echo
-			read -es -p "Retype new admin user password: " PASS2 ; echo
-			[ "x$PASS1" = "x$PASS2" ] || error "passwords do not match."
+			read -esr -p "Enter new admin user password: "  PASS1 ; echo
+			read -esr -p "Retype new admin user password: " PASS2 ; echo
```

Comment:
I really dont know what we expect here as default behavior. I personally prefer to not have to escape things if the input doesnt echo back what I currently type but this does limit the possible characters which are usable in normal usage.